apps/next/src/pages/Settings/components/NetworkManagement/components/Confirmations/Confirmation.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, Stack, Typography, useTheme } from '@avalabs/k2-alpine';
+import { PageTopBar } from '@/components/PageTopBar';
+import { MdInfoOutline } from 'react-icons/md';
+
+type ConfirmationProps = {
+  onBack: () => void;
+  onSubmit: () => void;
+  title: string;
+  warningText: string;
+  primaryButtonText: string;
+  secondaryButtonText: string;
+};
+
+export const Confirmation = ({
```

Comment:
Nice idea to make this generic enough so you can re-use it for all three use-cases 👏